src/ui/mathliveview.js

```diff
@@ -1,6 +1,6 @@
-import InputView from '@ckeditor/ckeditor5-ui/src/input/inputview';
+import { View } from 'ckeditor5/src/ui';
```

Comment:
FIx import, after changing `View` to `InputView` in https://github.com/bethinkpl/ckeditor5-math/pull/9/commits/2de8f651cfe49db8b4908bb22dd49854922ace9f

`InputView` automatically emits `input` event which is handled in https://github.com/bethinkpl/ckeditor5-math/blob/9bb53dd302deb01d281fdc80163c2a8ad6301c04/src/ui/mathliveview.js#L14 => we fetched value from mathlive even when mathlive didn't emit change